command-parser.js

```diff
@@ -30,6 +30,8 @@ const MESSAGE_COOLDOWN = 5 * 60 * 1000;
 
 const MAX_PARSE_RECURSION = 10;
 
+const VALID_COMMAND_TOKENS = ['/', '!'];
```

Comment:
Fair enough. I'd use a string and `.includes` rather than an array and `indexOf` though, but that's just habit.